### ISSUE TITLE
CI issue

### ISSUE THREAD

**Issue by vitaly-t** [vitaly-t/pg-promise#884]
GitHub Actions started showing problems recently, with the following error:

```
Connection for invalid connection must report the right error
   Message:
     Expected 'getaddrinfo EAI_AGAIN base' to contain 'password authentication failed for user'.
   Stacktrace:
=============================================================================
Writing coverage object [/home/runner/work/pg-promise/pg-promise/coverage/coverage.json]
     Error: Expected 'getaddrinfo EAI_AGAIN base' to contain 'password authentication failed for user'.
    at jasmine.Spec.<anonymous> (/home/runner/work/pg-promise/pg-promise/test/db.spec.js:264:43)
    at listOnTimeout (internal/timers.js:557:[17](https://github.com/vitaly-t/pg-promise/actions/runs/5098248657/jobs/9255178557#step:8:18))
    at processTimers (internal/timers.js:500:7)
```

This error has nothing to do with the project itself, it just appeared on its own, and relates to CI.

Need help finding the cause and fixing it. Any help is welcome!

---

@dplewis You originally integrated [CI to this project](https://github.com/vitaly-t/pg-promise/blob/master/.github/workflows/ci.yml), so maybe you know how to fix this?

**Comment by thezanke** [comment]
This appears to be an error being caused by a test in [db.spec.js](https://github.com/vitaly-t/pg-promise/blob/master/test/db.spec.js#L264). When `pgp('blah-blah')` gets called it's attempting to make a network connection; it expect there to be an error (in this case, not on windows or mac, so this is happening on linux & other systems).

The error the test expects to receive is one containing the text about "password authentication failure" but instead it's getting the "EAI_AGAIN" error, which is a DNS error in my experience. Does this happen every time you build or just intermittently? If it's every time, I would check your "Action Variables" section and make sure `POSTGRES_HOST` is set to "postgres" which is the label of the service in the CI action workflow.

**Comment by dplewis** [comment]
@vitaly-t Can you try to rerun the CI a few times to see if this issue still exists?

**Comment by vitaly-t** [comment]
Thank you everybody for your help, the issue is now resolved, and a new release 11.5.1 has been published.

The issue was, as @thezanke pointed, in a broken test, which weirdly, started failing for reasons outside this library. I had to amend an old test, in order to make it pass again.
